panic due to out-of-range access

Hello,

I encountered a panic during executing formatting.
Looks like there is out-of-range access to an array in the following line of code in the repository:
https://github.com/google/yamlfmt/blob/dd8547dbe27f5572ad2cfd3a794fb9ad454c1b23/internal/features/eof_newline.go#L31

Here is my log. Hope this helps.
```
$ yamlfmt ./
panic: runtime error: index out of range [-1]

goroutine 1 [running]:
github.com/google/yamlfmt/formatters/basic.ConfigureFeaturesFromConfig.MakeFeatureEOFNewline.eofNewlineFeature.func4({0x0?, 0x10bed95b8?, 0xa0?})
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/internal/features/eof_newline.go:31 +0xcc
github.com/google/yamlfmt.FeatureList.ApplyFeatures({0x1400007a740, 0x2, 0x104e4bf38?}, {0x0, 0x0, 0x104c44618?}, {0x104db43c8, 0x5})
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/feature.go:62 +0x194
github.com/google/yamlfmt/formatters/basic.(*BasicFormatter).Format(0x1400007a780, {0x14002100380?, 0x104e37880?, 0x14000407a40?})
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/formatters/basic/formatter.go:81 +0x42c
github.com/google/yamlfmt/engine.(*ConsecutiveEngine).FormatContent(...)
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/engine/consecutive_engine.go:34
github.com/google/yamlfmt/engine.(*ConsecutiveEngine).formatFileContent(0x14000947db8, {0x140003a7710, 0x28})
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/engine/consecutive_engine.go:91 +0x64
github.com/google/yamlfmt/engine.(*ConsecutiveEngine).formatAll(0x14000947db8, {0x14000154008, 0x3eb, 0x1400000e210?})
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/engine/consecutive_engine.go:76 +0xac
github.com/google/yamlfmt/engine.(*ConsecutiveEngine).Format(0x14000947db8, {0x14000154008?, 0x3eb?, 0x3ff?})
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/engine/consecutive_engine.go:38 +0x24
github.com/google/yamlfmt/command.(*Command).Run(0x14000219e98)
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/command/command.go:127 +0x3f8
main.run()
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/cmd/yamlfmt/main.go:79 +0x314
main.main()
        /Users/shion/go/pkg/mod/github.com/google/yamlfmt@v0.13.0/cmd/yamlfmt/main.go:35 +0x1c
```

--- a/internal/features/eof_newline.go
+++ b/internal/features/eof_newline.go
@@ -28,7 +28,7 @@ func MakeFeatureEOFNewline(linebreakStr string) yamlfmt.Feature {
 func eofNewlineFeature(linebreakStr string) yamlfmt.FeatureFunc {
 	return func(content []byte) ([]byte, error) {
 		// This check works in both linebreak modes.
-		if content[len(content)-1] != '\n' {
+		if len(content) == 0 || content[len(content)-1] != '\n' {
 			linebreakBytes := []byte(linebreakStr)
 			content = append(content, linebreakBytes...)
 		}

